CHANGELOG.md

```diff
@@ -1,5 +1,398 @@
+- Update Android Gradle plugin to 3.6.4 ([cf8368f204](https://github.com/facebook/react-native/commit/cf8368f2046ae1ff0f6b02bb6857eeeff8f57d7d) by [@friederbluemle](https://github.com/friederbluemle))
+- Update Gradle Wrapper to 6.6 ([5bc67b658e](https://github.com/facebook/react-native/commit/5bc67b658e581e0176deb7ed95b51a5c1cbe65c2) by [@friederbluemle](https://github.com/friederbluemle))
+- Upgraded to Hermes 0.6.0 ([776a415d98](https://github.com/facebook/react-native/commit/776a415d98dffd04b11200812a32204aa1c5e157) by [@Huxpro](https://github.com/Huxpro))
+- Update Android build tools to 29.0.3 ([e629e94b46](https://github.com/facebook/react-native/commit/e629e94b466ebbd5924b1d4493c026004dad707d) by [@friederbluemle](https://github.com/friederbluemle))
+- Bump Gradle to 6.5 ([e559aee642](https://github.com/facebook/react-native/commit/e559aee64275126eaa135486e6bf09138be70f4d) by [@dulmandakh](https://github.com/dulmandakh))
+- ViewCommands on Android now execute earlier, as a perf optimization. ([c6b9cc36da](https://github.com/facebook/react-native/commit/c6b9cc36da4f7d190d05122048aa4ada9c152b73) by [@JoshuaGross](https://github.com/JoshuaGross))
+- Effect of `blurRadius` now more closely matches other platforms. ([64860972be](https://github.com/facebook/react-native/commit/64860972be828fb601acbef11b4c2dbc672dee8a) by [@yungsters](https://github.com/yungsters))
+- Moved TimePickerAndroid to FB internal. ([c8fed9e385](https://github.com/facebook/react-native/commit/c8fed9e3858876c4e9cfe452a9f51c9241f82aad))
+- Test Modernization ([6a78b32878](https://github.com/facebook/react-native/commit/6a78b32878aea1b0dac98ff36378fb9392d4aeb1))
+- Renamed EventDispatcher to EventDispatcherImpl and created EventDispatcher interface; calls to EventDispatcher contstructor need to be updated ([68c0eddb71](https://github.com/facebook/react-native/commit/68c0eddb71a5bbc44da980b82c7e718b454f0267) by [@ejanzer](https://github.com/ejanzer))
+- Get ripple drawables by id ([c8ed2dbbb2](https://github.com/facebook/react-native/commit/c8ed2dbbb287deed05a8782fb8665c1edf45bbac) by [@vonovak](https://github.com/vonovak))
+- Internal change to make `PlatformConstants` use the same method to determine `ServerHost` as other code paths ([d20ac69fc8](https://github.com/facebook/react-native/commit/d20ac69fc87ef4f9f5921356c6bbfc0587a9375c) by [@ide](https://github.com/ide))
+- Make CatalystInstance.getNativeModule nullable ([1cef72af04](https://github.com/facebook/react-native/commit/1cef72af047bd2e56d774858093f1ee6269c2d27) by [@RSNara](https://github.com/RSNara))
+- ScrollView now supports `contentOffset` ([ed29ba13f9](https://github.com/facebook/react-native/commit/ed29ba13f97f240c91fdf6c0ef3fb601046697b9) by [@JoshuaGross](https://github.com/JoshuaGross))
+- Update Gradle Wrapper to 6.3 ([8988a073b4](https://github.com/facebook/react-native/commit/8988a073b48df0f0cd4a7126edf1a421f4537d58) by [@friederbluemle](https://github.com/friederbluemle))
```

Comment:
Seems like the gradle wrapper and plugin changes could be coalesced, not necessary to mention that during the dev cycle it went to 6.3 then 6.5 then 6.6 etc ?